lib/OnboardingMessageProvider.jsm

```diff
@@ -69,12 +71,30 @@ const ONBOARDING_MESSAGES = () => ([
       button_label: {string_id: "onboarding-button-label-try-now"},
       button_action: {
         type: "OPEN_URL",
-        data: {url: "https://addons.mozilla.org/en-US/firefox/addon/ghostery/"},
+        data: {args: "https://addons.mozilla.org/en-US/firefox/addon/ghostery/"},
       },
     },
     targeting: "providerCohorts.onboarding == 'ghostery'",
     trigger: {id: "firstRun"},
   },
+  {
+    id: "ONBOARDING_5",
+    template: "onboarding",
+    bundled: 3,
+    order: 4,
+    content: {
+      title: {string_id: "onboarding-fxa-title"},
+      text: {string_id: "onboarding-fxa-text"},
+      icon: "sync",
+      button_label: {string_id: "onboarding-button-label-get-started"},
+      button_action: {
+        type: "OPEN_URL",
+        data: {args: "https://accounts.firefox.com/"},
```

Comment:
Note: this URL is actually not the correct url that it should point to, but I need to talk to @vladikoff about what the proper url will be... we can do it as a follow up since this won't be live for a bit

Reply:
Yup I'll add a comment, and I'll file a bug 👍

Reply:
Bug filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1503912

Reply:
I'm going to close bug 1503912 because i just added the correct url as part of this pr